README.org

```diff
@@ -107,6 +107,19 @@ To enable, customize =consult-dir-project-list-function= or
 (setq consult-dir-project-list-function #'consult-dir-projectile-dirs)
 #+END_SRC
 
+*** Remote hosts
```

Comment:
Not enabled by default, the user can customize `consult-dir-sources`, see below.